### PR TITLE
Improve documentation for local setup

### LINKED ISSUE #20: MVD local machine setup for devs
Feature #53 

## Description

Enable MVD to run on local developer computers of to enable efficient development and debugging of EDC connectors

## Acceptance Criteria

- [x] MVD components can be deployed to an own Azure Sub (KeyVaults, Storage Accounts, etc)
- [x] EDC connectors can be run on local machine
- [x] Instructions how to set up local development machine
- [x] Instructions how to debug EDC connectors 
- [ ] ~A Postman suite can be used to issue requests to EDC connectors~ moved to #159

## Tasks
- [x] Enable running system tests locally
- [x] Enable debugging

### LINKED ISSUE #121: MVD local machine setup for devs

### PR DESCRIPTION
**Pull request by marcgs**
closes agera-edc/MinimumViableDataspaceFork#121 

Extended documentation for local development setup

### CODE REVIEW

File: system-tests/README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 ## System tests
 
-The test use the key vault secret to connect to the storage accounts and copy a file
-from provider to consumer storage account.
+The test use the key vault secret to connect to the storage accounts and copy a file from provider to consumer storage account.
```
Comment (ouphi) [inline system-tests/README.md]:
```suggestion
The test uses the key vault secret to connect to the storage accounts and copy a file from provider to consumer storage account.
```

File: system-tests/README.md [hunk 2]
```diff
@@ -44,4 +42,8 @@ CONSUMER_KEY_VAULT="$CONSUMER_KEY_VAULT" ./gradlew :system-tests:test
 
 Follow the instructions in the previous sections to run an MVD with a consumer and provider locally using docker-compose. 
 
-Once running, you can use a Java debugger to connect to the consumer (port 5006) and provider (port 5005) instances. If you are using IntelliJ you can use the provided "EDC consumer" or "EDC provider" runtime configurations to remote debug the connector instances. 
\ No newline at end of file
+Once running, you can use a Java debugger to connect to the consumer (port 5006) and provider (port 5005) instances. If you are using IntelliJ you can use the provided "EDC consumer" or "EDC provider" runtime configurations to remote debug the connector instances.
```
Comment (ouphi) [inline system-tests/README.md]: nit: When mentioning the "runtime configurations", you could add a link to the .run folder

**Review by ouphi — APPROVED** [review]

File: system-tests/README.md (hunk 2 above)
Comment (marcgs) [reply system-tests/README.md]: These are automatically imported by IntelliJ, I am not sure that would help

Comment (marcgs) [reply system-tests/README.md]: But since this was already mentioned by @cpeeyush already I can add the link... it does not hurt either

**Review by cpeeyush — APPROVED** [review]

**Pull request merged**